packages/taquito/test/estimate/rpc-estimate-provider.spec.ts

```diff
@@ -26,7 +26,9 @@ import {
 } from '../contract/helper';
 import { OpKind } from '@taquito/rpc';
 import { TransferTicketParams } from '../../src/operations/types';
-import { InvalidAddressError, InvalidContractAddressError } from '@taquito/utils';
+import { InvalidContractAddressError } from '@taquito/utils';
```

Comment:
I think this can be moved/extended from `@taquito/core` as it's basically a parameter error or a `InvalidXError` but perhaps that's for the other ticket

Reply:
i have moved it to taquito core!

Reply:
will push soon